README.md

```diff
@@ -3,10 +3,10 @@
 The Kani Rust Verifier is a bit-precise model checker for Rust.
 
 Kani verifies:
- * memory safety -- even in unsafe Rust code
- * user-specified assertions
- * absence of panics
- * checks for some classes of undefined behavior
+ * Memory Safety -- even in unsafe rust code.
+ * User specified assertions.
+ * Absence of panics.
+ * Checks for some classes of undefined behavior.
```

Comment:
In case it's helpful, the rule I followed to write this is the second case in https://www.dailywritingtips.com/bulleted-lists/

I'm okay with first letters being capitalized, but I don't think we should keep the dots. Also, Rust is a name.

Reply:
Thanks for the link, I'll use it here on. I think keeping the first letters capitalized makes it easier to read. I'll remove the dots add Rust as a name.